feat(lb): add ipv6 doc (#2595)
scaleway/docs-content: faq/loadbalancer.mdx

## What is a flexible IP address?

Your frontend listens to your Load Balancer's public flexible IP address. In case of a failure of the Load Balancer, a replica Load Balancer is immediately spawned and deployed, and the IP address is automatically rerouted to this replica. This is done automatically, by the Load Balancer control subsystems.

By default, each [public](/network/load-balancer/concepts/#accessibility) Load Balancer is created automatically with a flexible IPv4 address. You can optionally also add an IPv6 address.

When you delete a Load Balancer, you can choose to keep its flexible IP(s) in your account, to reuse later with a new Load Balancer. These 
flexible IPs are not compatible with other Scaleway products (e.g. Instances, Elastic Metal servers, Public Gateways): each resource has its own set of flexible IPs.

## Can I have more than one flexible IP address for a Load Balancer?

> Each Load Balancer can have one public IPv4 address and one public IPv6 address. Currently, it is not possible to assign more than one of each type of IP to a given Load Balancer.

Reviewer: ```suggestion
Each Load Balancer can have one public IPv4 address and one public IPv6 address. Currently, it is not possible to assign more than one of each IP type to a given Load Balancer.
```

## Do Load Balancers support external IPv6 connections?

> Yes, Load Balancer supports both IPv4 and IPv6 addresses at the frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.

Reviewer 2: ```suggestion
Yes, Load Balancers support both IPv4 and IPv6 addresses at the frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.
```

Reviewer 2: Just to be consistent with the plural used in the question ffti

## Can I have more than one flexible IP address for a Load Balancer?

Each Load Balancer can have one public IPv4 address and one public IPv6 address. Currently, it is not possible to assign more than one of each type of IP to a given Load Balancer.

</Concept>

<Concept>

## Do Load Balancers support external IPv6 connections?

> Yes, Load Balancer supports both IPv4 and IPv6 addresses at the public frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.

Author: ```suggestion
Yes, Load Balancer supports both IPv4 and IPv6 addresses at the frontend. IPv6 can also be used to communicate between the Load Balancer and your backend servers.
```
